sdk-python/test/check_calc_keys_test.py

```diff
@@ -1,17 +1,24 @@
 import unittest
 from typing import Any
+from unittest.mock import patch
 
 import pytest
 
 from omnilogger.pricing.pricing_class import Pricing
+from omnilogger.pricing.storage_manager import pricing_reference_path
```

Comment:
Wrote a comment but i cant see it anymore! not sure what happened - but was wondering if its worth moving this into the test functionality? Since it's not something used in the app logic and could be confusing

Reply:
It is used to locate the default pricing's path in the self.__init_llm_pricing_data